README.md

```diff
@@ -49,17 +49,65 @@ There is no return value from this method.
 
 If language selection fails (no matching languages), the callback will contain an error.
 
+
+#### `.transformByField({source}, [languages], {fields});`
+
+This method applies the transform on a field by field basis as opposed to comparing a whole `i18n`
+block like the `transform` method does.
+
+The `fields` object should adhere to the following schema:
+
+```
+{
+    "required": [
```

Comment:
Quotes - They just provide a bit of highlighting in markdown... No big deal, I can remove them.

Names - I do agree with the names no being clear on the intent, not sure fallback & nonFallback seem better.. Still not sure they are clear though. Lets roll with your suggestion for the moment though. 

What if a field is not specified in either list? - If a field is not present in either list then it doesn't get queried for... I'm not sure i'm following what you are asking for?